src/Widgets/ScheduledRow.vala

```diff
@@ -0,0 +1,63 @@
+/*
+* Copyright 2019 elementary, Inc. (https://elementary.io)
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 3 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+*/
+
+public class Tasks.ScheduledRow : Gtk.ListBoxRow {
+
+    private Gtk.Image icon;
+    private Gtk.Image status_image;
+    private Gtk.Label display_name_label;
+    private Gtk.Stack status_stack;
+    private Gtk.Revealer revealer;
```

Comment:
These probably don't all need to be declared ahead of time since they're only used in the construct block

Reply:
Fixed.